src/sample/data/student1courselist.txt

```diff
@@ -0,0 +1,25 @@
+CPT111 - PRINCIPLES OF PROGRAMMING
+CPT112 - DISCRETE STRUCTURES
+CST131 - COMPUTER ORGANISATIONS
+CPT113 - PROGRAMMING METHODOLOGY AND DATA STRUCTURES
+CPT115 - MATHEMATICAL METHODS FOR COMPUTER SCIENCE
+CPC151 - FUNDAMENTALS OF LOGIC AND ARTIFICIAL INTELLIGENCE
+CAT201 - INTEGRATED SOFTWARE DEVELOPMENT WORKSHOP
+CMT221 - DATABASE ORGANISATIONS AND DESIGN
+CST232 - OPERATING SYSTEMS
+CSE241 - FOUNDATION OF SOFTWARE ENGINEERING
+AKW103 - INTRODUCTION TO MANAGEMENT
+AKW104 - ACCOUNTING AND FINANCE
+ATW202 - BUSINESS RESEARCH METHOD
+ATW241 - PRINCIPLES OF MARKETING
+ATW262 - PRINCIPLES OF FINANCE
+MAA101 - CALCULUS FOR SCIENCE STUDENTS
+MAT111 - LINEAR ALGEBRA
+MAT181 - PROGRAMMING FOR SCIENTIFIC APPLICATIONS
+MAA111 - ALGEBRA FOR SCIENCE STUDENTS
+LAK100 - KOREAN LANGUAGE
+LAJ100 - JAPANESE LANGUAGE
+LKM400 - BAHASA MALAYSIA
+LSP300 - ACADEMIC ENGLISH
+HFF225 - PHILOSOPHY AND CURRENT ISSUES
+HFE224 - APPRECIATION OF ETHICS AND CIVILISATIONS
```

Comment:
why there are 2 course lists, `courseList.txt ` and `student1courselist.txt `? If they serve the same purpose why don't only reuse the course list (Registered course is another thing larh)

Reply:
@Michelle-Lohwt i think according to @GitHubUser5136 , the courselist.txt is to display the course list while student1courselist.txt is to see what course have been registered by student 1, @GitHubUser5136 correct me if i am wrong

Reply:
![image](https://user-images.githubusercontent.com/94297164/149626464-5fa8cd4a-fd02-4d9f-886b-79e6e33d94e0.png)
The difference between `courseList.txt` and `student1courselist.txt`